linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/rule.yml

```diff
@@ -25,13 +25,14 @@ references:
     cis: 5.2.12
     cjis: 5.5.6
     cui: 3.1.11
-    disa: 1133,2361
+    disa: 879,1133,2361
     hipaa: 164.308(a)(4)(i),164.308(b)(1),164.308(b)(3),164.310(b),164.312(e)(1),164.312(e)(2)(ii)
-    nist: AC-2(5),SA-8,AC-12,AC-17(b)
+    nist: AC-2(5),SA-8,AC-12,AC-17(b),MA-4(e),SC-10,AC-12
```

Comment:
MA-4(e) is about terminating sessions when work is completed, not when idle timeouts occur. Can you help us understand the reasoning behind adding the MA-4(e) mapping?